_includes/api/en/4x/req-fresh.md

```diff
@@ -2,12 +2,19 @@
 
 When the response is still "fresh" in the client's cache `true` is returned, otherwise `false` is returned to indicate that the client cache is now stale and the full response should be sent.
 
+In order to check whether the response is still "fresh" in the client's cache or not, two tags will be used ETag and If-None-Match. ETag will be fetched from response (saved internally from application) and other tag will be received from client's request.
```

Comment:
The headers which go into determing fresh are seen here in [jshttp/fresh](https://github.com/jshttp/fresh/blob/206d8c5dbe5022d05f5473481852e393bbb2194e/index.js#L33C1-L85C2)

* If-None-Match
* Cache-Control: no-cache
* If-Modified-Sense
* Etag